src/App.css

```diff
@@ -208,6 +210,7 @@ html, body {
 	text-decoration: none;
   text-shadow: 0px 1px 0px var(--alt);
   margin-right: 16px;
+  margin-bottom: 32px;
```

Comment:
I added some margin to the bottom of the buttons so they never touch the end of the screen.